src/components/file-preview.vue

```diff
@@ -123,21 +158,37 @@ export default {
         },
         initOptions: {
           background: false,
-          viewMode: 0,
+          viewMode: 1,
           autoCropArea: 1,
           zoomable: false
         }
       }
     };
   },
-  watch: {
-    "image.cropRatio"(newValue) {
-      this.setAspectRatio(newValue);
-    }
-  },
   computed: {
+    componentId() {
+      return "file-preview-" + ++id;
```

Comment:
Instead of using a globally tracked numerical ID, lets rely on shortid instead:

```suggestion
      return "file-preview-" + this.$helpers.shortid.generate();
```

Reply:
Perfect, that one I did in the most straightforward manner, bc I knew that should be better way of handling such cases in this project. Actually I see that in google old directus docs page about `contribution - helpers` appears - why it's not part of current docs anymore?

Reply:
You mean this page? https://docs.directus.io/advanced/app/helpers.html

Reply:
o.O seems like google has cached old url for it - https://www.google.pl/search?q=directus+helpers

Nevertheless - from which page helpers docs are accessible? I cannot find a link on contributing page

Reply:
Oh good question.. I always use the search function 😁 @benhaynes ?

Reply:
Maybe that's why search crawlers didn't update link as well 😃

Reply:
Fixed! https://docs.directus.io/advanced/app/helpers.html